### PR TITLE
Add metadata options to VM launch configuration

### PR DESCRIPTION
**Pull request by aaron-lane**
## Background

When using an IAM instance profile to authenticate Cost Estimation for AWS resources, a metadata HTTP PUT response hop limit of at least 2 is required to account for the use of IMDSv1.

## How Has This Been Tested

Deployed and verified that Cost Estimation worked for AWS resources using an IAM instance profile.

## This PR makes me feel

optional gif describing your feelings about this pr

### CODE REVIEW

**Review by bnferguson — APPROVED** [review]

**Comment by aaron-lane** [comment]
@bnferguson I dismissed your review to fix formatting 🙇🏼

**Review by bnferguson — APPROVED** [review]
Have another review! 🎉  :shipit:

**Review by anniehedgpeth — APPROVED** [review]

**Pull request merged**
